scripts/pyproject.toml

```diff
@@ -0,0 +1,108 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "scverse-template-scripts"
+dynamic = ["version"]
+description = "scripts for ecosystem package data"
+readme = "../README.md"
+requires-python = ">=3.11"
+license = "GPL-3.0"
+authors = [
+    { name = "Philipp A.", email = "flying-sheep@web.de" },
+]
+urls.Documentation = "https://github.com/scverse/cookiecutter-scverse#readme"
+urls.Issues = "https://github.com/scverse/cookiecutter-scverse/issues"
+urls.Source = "https://github.com/scverse/cookiecutter-scverse"
+dependencies = [
+    "rich",
+    "typer",
+    "furl",
+    "GitPython",
+    "PyGitHub",
+    "PyYAML",
+]
+
+[project.optional-dependencies]
+test = [
+    "pytest",
+    "pytest-git",
+    "pytest-network",
+    "pytest-responsemock",
+]
+
+[project.scripts]
+send-cruft-prs = "scverse_template_scripts.cruft_prs:cli"
+make-rich-output = "scverse_template_scripts.make_rich_output:main"
+
+[tool.hatch.version]
+source = "vcs"
+fallback-version = "0.0"
+
+[tool.hatch.envs.default]
+python = "3.11"
+
+[tool.hatch.envs.test]
+features = ["test"]
+[tool.hatch.envs.test.scripts]
+test = "pytest"
+
+[tool.pytest.ini_options]
+addopts = [
+    "--import-mode=importlib",
+    "-pgit",
+    "-pnetwork", "--disable-network",
+    "-presponsemock"
+]
```

Comment:
why is this needed?

Reply:
The `-p???` switches activate the pytest plugins used and make pytest complain if they’re missing. The `--disable-network` switch makes sure all network requests are mocked out.